CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_google_ads v0.13.0
+
+## Schema Updates
+
+**2 total changes • 1 possible breaking change**
+| **Model/Column** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| [google_ads__search_term_report](https://fivetran.github.io/dbt_google_ads/#!/model/model.google_ads.google_ads__search_term_report) | New Column |   |  `criterion_id` | **BREAKING:** This may change the model's grain, as a single `keyword_text` can have multiple `criterion_id` values. `criterion_id` is included in uniqueness tests for this model. |
+| [stg_google_ads__search_term_keyword_stats](https://fivetran.github.io/dbt_google_ads_source/#!/model/model.google_ads_source.stg_google_ads__search_term_keyword_stats) | New Column |   |  `criterion_id` |  Parsed out of `keyword_ad_group_criterion` field |
+
+## Feature Updates
+- Added `criterion_id` to the recently introduced `google_ads__search_term_report` model. This is parsed from the source `keyword_ad_group_criterion` table to extract the ID of the associated keyword. 
+  - A single `keyword_text` can be associated with multiple `criterion_id` values, so this may impact the grain/volume of the end model and has therefore been included in model uniqueness tests. 
+  - This was added to align with other advertising platforms in the downstream [Ad Reporting](https://github.com/fivetran/dbt_ad_reporting/tree/main?tab=readme-ov-file) data model, which includes keyword IDs in the combined [search report](https://fivetran.github.io/dbt_ad_reporting/#!/model/model.ad_reporting.ad_reporting__search_report) end model.
```

Comment:
These are a bit duplicative to what's already mentioned in the schema change table. Not partial to including or removing, but wanted to provide the suggestion that we can probably consolidate and use the table to inform these changes.

Reply:
I'll consolidate but keep the Feature Updates section for the upstream Ad Reporting note